components/tyk-gateway/templates/deployment-gw-repset.yaml

```diff
@@ -135,12 +137,19 @@ spec:
             value: "/opt/tyk-gateway/templates"
           - name: TYK_GW_TYKJSPATH
             value: "/opt/tyk-gateway/js/tyk.js"
+        {{- if .Values.gateway.scratch.enabled }}
           - name: TYK_GW_MIDDLEWAREPATH
             value: "/mnt/tyk-gateway/middleware"
           - name: TYK_GW_APPPATH
             value: "/mnt/tyk-gateway/apps"
           - name: TYK_GW_POLICIES_POLICYPATH
             value: "/mnt/tyk-gateway/policies"
+        {{- else}}
```

Comment:
In this part, i guess `TYK_GW_POLICIES_POLICYPATH` is missing.

Reply:
it's on purpose, seem my previous comment  https://github.com/TykTechnologies/tyk-charts/pull/108/files/109f8f1d2e22ed4391f4a2141b7f948286d4b408#r1310550140

Reply:
I see your explanation; however, I am seeking further clarification regarding the rationale behind not setting it. Would you kindly provide additional details or elaborate on the reasoning behind the decision?

Reply:
It's covered by TYK_GW_POLICIES_POLICYRECORDNAME